### DISCUSSION TITLE
[Request] Papermerge

### DISCUSSION THREAD

**Discussion by FroehlicherFuchs** [tteck/Proxmox#3167] [Ideas]
Hi, I found a new cool DMS. Maybe there's a LXC possible. 

https://papermerge.com/
https://github.com/ciur/papermerge

Thank you!!

**Comment by smignon612** [comment]
paperless is al ready avail. ;)

**Reply by FroehlicherFuchs** [reply]
I mean Papermerge not paperless :-)
